src/Form/TfaLoginForm.php

```diff
@@ -164,11 +164,6 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       }
       elseif ($this->ready($tfaValidationPlugin) && !$this->loginAllowed($account)) {
 
-        // Restart flood levels, session context, and TFA process.
-        // flood_clear_event('tfa_validate');
```

Comment:
Yes. Before there two flood controls `tfa_begin` and `tfa_validate`, the former was triggered after doing login through username and password and the later after a failed Two Factor attempt.
`tfa_begin` flood control did not make much sense to me so I removed it and shifted `tfa_validate` to `EntryForm.php` where validation is handled and changed the name to `tfa.failed_validation` based on the guidelines for naming these.